packages/react/.storybook/webpack.config.js

```diff
@@ -138,5 +138,9 @@ module.exports = ({ config, mode }) => {
     plugins: [new FeatureFlagProxyPlugin()],
   };
 
+  config.plugins = config.plugins.filter(
+    p => String(p.resourceRegExp) !== '/core-js/'
```

Comment:
What is this line for? Sounds that you hit an issue with original WebPack config that is worth putting a special note here?